client/src/components/DeidentificationConfigForm.js

```diff
@@ -90,62 +90,62 @@ export class DeidentificationConfigForm extends React.Component {
     const allAnnotationTypes = Object.values(DeidentificationStepAnnotationTypesEnum)
     return (
       <Collapse in={this.state.expand}>
-      <div className="deid-config-form">
-        <div className="deid-config-form-bar">
-          <div className="deid-config-header">De-id Step #{this.props.index + 1}</div>
-          <div className="deid-config-remove" onClick={this.handleDelete}></div>
-        </div>
-        <table>
-          <tr>
-            <td>
-              Method
-            </td>
-            <td>
-              <select onChange={this.handleStrategyChange} value={this.getStrategy()}>
-                {Object.keys(DEIDENTIFICATION_STRATEGIES).map((strategy) => {
-                  return <option value={strategy} key={strategy}>{DEIDENTIFICATION_STRATEGIES[strategy]}</option>;
-                })}
-              </select>
-              &nbsp;
-              {this.getStrategy() === "maskingCharConfig" &&
-                <input type="text" maxLength={1} value={this.props.maskingCharConfig.maskingChar} onChange={this.handleMaskingCharChange} />
-              }
-            </td>
-          </tr>
-          <tr>
+        <Paper style={{ backgroundColor: "grey", color: "white", fontFamily: "monospace", fontSize: "16px" }}>
+          <div className="deid-config-form-bar">
+            <div className="deid-config-header">De-id Step #{this.props.index + 1}</div>
+            <div className="deid-config-remove" onClick={this.handleDelete}></div>
+          </div>
+          <table>
+            <tr>
+              <td>
+                Obfuscation Method
```

Comment:
To stay consistent with other option names.

```suggestion
                Obfuscation method
```